notifier/management/commands/forums_digest_flagged.py

```diff
@@ -0,0 +1,84 @@
+"""
+Send a digest list of threads with flagged posts to forum moderators.
+"""
+from optparse import make_option
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from notifier.digest import render_digest_flagged
+from notifier.pull import get_flagged_threads
+from notifier.tasks import do_forums_digests_flagged
+from notifier.user import get_moderators
+
+
+class Command(BaseCommand):
+    """
+    This Command is used to send a digest of threads with flagged posts to
+    forum moderators.
+    """
+
+    help = "Send a digest list of threads with flagged posts to each moderator"
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--is-dry-run',
+            action='store_true',
+            dest='is_dry_run',
+            default=False,
+            help='for each course with flagged posts, display the email address'
+                 ' for each recipient, and the rendered email text WITHOUT'
+                 ' sending a message',
+        ),
+    )
+
+    def dry_run(self):
+        """
+        Print the recipients and rendered email text for each course with flagged posts.
+        """
+        course_threads = {}
+        for thread in get_flagged_threads():
+            course_id = thread.get('course_id')
+            commentable_id = thread.get('commentable_id')
+            comment_thread_id = thread.get('comment_thread_id')
+            if course_id and commentable_id and comment_thread_id:
+                if course_id not in course_threads:
+                    course_threads[course_id] = []
+                thread_url = '{base_url}/courses/{course_id}/discussion/forum/{commentable_id}/threads/{comment_thread_id}'.format(
+                    base_url=settings.LMS_URL_BASE,
+                    course_id=course_id,
+                    commentable_id=commentable_id,
+                    comment_thread_id=comment_thread_id,
+                )
+                course_threads[course_id].append(thread_url)
+
+        for course_id in course_threads:
+            moderators = get_moderators(course_id)
+            try:
```

Comment:
Can we add a quick note here explaining why we're doing this?

I understand it now that you explained it to me, but it's not immediately clear what's going on.

Something like:

```python
# We only grab the first moderator because we're only concerned with rendering a single sample message.
# The assumption is that each message will be similar enough for testing purposes.
```